### DISCUSSION TITLE
How to disable instrospection and GraphiQL?

### DISCUSSION THREAD

**Discussion by ghost** [Netflix/dgs-framework#710] [Q&A] [answered]
Hello Everyone

Is there any way to disable (in production for security reasons) the schema introspection
`{
  __schema {
    types {
      name
    }
  }
}`

and also to disable the GraphiQL URL for the same reason? Because I cannot find how to do it.

Thank you very much.

**Comment by paulbakker** [comment, marked as answer]
There are config properties for that: https://netflix.github.io/dgs/configuration/

**Reply by aisensiy** [reply]
I think this doc miss an import configuration: `dgs.graphql.introspection.enabled: false`. If this is still enabled it will not stop people to get the full schema: https://www.apollographql.com/docs/rover/graphs/#fetching-via-introspection

**Reply by aisensiy** [reply]
https://github.com/Netflix/dgs/pull/81

**Reply by srinivasankavitha** [reply]
Thanks for the doc update! Merged.
